ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.ts

```diff
@@ -558,21 +558,27 @@ export class AddressEmbeddedComponent implements OnDestroy, OnInit {
             tap(address => this.inputAddress$.next(address))
         );
 
-        const emitComponentBusyAction$ = combineLatest([this.isInputComponentBusy$, isThisComponentBusy$]).pipe(
+        const addressComponentBusy$ = combineLatest([this.isInputComponentBusy$, isThisComponentBusy$]).pipe(
             map(busyFlags => busyFlags.some(val => val)),
             distinctUntilChanged(),
+            share());
+
+        const emitComponentBusyAction$ = addressComponentBusy$.pipe(
+            // let's smooth out our busyness signals
+            debounce(busy => interval(busy ? 0 : 250)),
+            distinctUntilChanged(),
             tap(isBusy => this.componentBusy.emit(isBusy))
         );
```

Comment:
If we are busy, we want to report that right away. If we are not busy, we want to wait just a little bit before reporting just in case we become busy again.
distinctUntilChanged just keeps the number of emissions to only changes between busy and not busy.